Example/FinancialConnections Example/FinancialConnections Example/Playground/PlaygroundMainView.swift

```diff
@@ -16,50 +16,65 @@ struct PlaygroundMainView: View {
         ZStack {
             VStack {
                 Form {
-                    VStack(alignment: .leading, spacing: 4) {
-                        Text("What do you want to use Financial Connections for?")
-                            .font(.headline)
-                        Picker("Flow?", selection: $viewModel.flow) {
-                            ForEach(PlaygroundMainViewModel.Flow.allCases) {
-                                Text($0.rawValue.capitalized)
-                                    .tag($0)
+                    Section {
```

Comment:
all of this code does two things:
1. Adds public key and sercret key fields to playground app
2. Automatically shows/hides some of the fields depending on what is selected (for example, if app to app is enabled, one can't toggle test mode)